pkgs/applications/audio/gtkcord4/default.nix

```diff
@@ -38,19 +41,27 @@ buildGoModule rec {
     graphene
     gtk4
     pango
+    # Optional according to upstream but required for sound and video
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-ugly
+    gst_all_1.gstreamer
+    libcanberra-gtk3
+    sound-theme-freedesktop
   ] ++ lib.optionals withLibadwaita [
     libadwaita
   ];
 
   tags = lib.optionals withLibadwaita [ "libadwaita" ];
 
   postInstall = ''
-    install -D -m 444 -t $out/share/applications .nix/com.github.diamondburned.gtkcord4.desktop
+    install -D -m 444 -t $out/share/applications nix/xyz.diamondb.gtkcord4.desktop
```

Comment:
unclear what this is doing, but I trust you tested it and it was not working before?

Reply:
That's due to this upstream change https://github.com/diamondburned/gtkcord4/pull/102, which was necessary for FlatHub compatibility.